packages/lesswrong/components/tagging/LWTagPage.tsx

```diff
@@ -246,6 +250,29 @@ const styles = defineStyles("LWTagPage", (theme: ThemeType) => ({
     width: 18,
     marginRight: 4,
   },
+  nonMobileAudioPlayer: {
+    display: 'flex',
+    position: 'absolute',
+    right: 8,
+    width: AUDIO_PLAYER_WIDTH,
+    [theme.breakpoints.down('sm')]: {
+      display: 'none !important',
```

Comment:
I'm a little worried about all the `!important`s I see in this PR, but I assume some of them are working around issues with the audio player being a 3rd-party thing?

Reply:
These were not necessary and have been removed